common/filter-tags/README.md

```diff
@@ -0,0 +1,47 @@
+# FILTER TAGS Datadog Generator
+
+## How to use this module
+
+This module usage should be transparent because it should be used inside each detectors modules directly.
+Here is a simple example but it is advisable to see how are created other existing detectors modules:
+
+```
+module "filter-tags" {
+  source = "../../common/filter-tags"
```

Comment:
no it depends where your module is located in repo so I cannot provide a generic answer to that.

do you prefer I use a repo link instead of a local address ?